scripts/build-single-installer-win64.bat

```diff
@@ -0,0 +1,62 @@
+set OS=win64
+set CLUSTER=%1
+
+@echo ##############################################################################
+@echo ###
+@echo ### Building installers:  %CLUSTER%
+@echo ###
+@echo ##############################################################################
+
+pushd config
+    set LAUNCHER="./launcher.dhall ( ./launcher-%OS%.dhall ./%CLUSTER%.dhall ) "
+    set TOPOLOGY="./topology.dhall                         ./%CLUSTER%.dhall"
+    @echo ##############################################################################
+    @echo ###
+    @echo ### Generating configs:  %CLUSTER%
+    @echo ###
+    @echo ###  LAUNCHER=%LAUNCHER:"=%
+    @echo ###  TOPOLOGY=%TOPOLOGY:"=%
+    @echo ###
+    @echo ##############################################################################
+    del /f ..\installers\launcher-config.yaml 2>nul
+    del /f ..\installers\wallet-topology.yaml 2>nul
+
+    echo %LAUNCHER:"=% | call dhall-to-yaml > ..\installers\launcher-config.yaml
```

Comment:
Why is this not part of haskell code? It could be there united for both platforms, not having to maintain .bat files

Reply:
It's a good step, I agree.